specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,52 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid) which are not captured in another FOCUS column. These properties enable a practitioner to more easily understand the differences that make up the multiple **SKU Price ID**s under a given [**SKU ID**](#skuid).
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The properties (both key and value) contained in the _SkuPriceDetails_ column MUST be shared across all resources having the same _SkuPriceId_.
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are already captured in the [_SkuDetails_](#skudetails) column.
```

Comment:
FYI - I don't have a way to do this at scale. I get it's a SHOULD NOT, which is fine, but I'm calling it out because I don't see how this will ever be scalable without a significant change across hundreds of teams. I'm assuming other providers are the same.

Reply:
ACK. Leaving this comment  open for when I try to put together supporting content.

Reply:
This is redundant as this clause was eliminated when SKU Details was eliminated in favor of only having the SKU Price Details column.